src/main/webapp/WEB-INF/driver/all.jsp

```diff
@@ -0,0 +1,33 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All drivers</title>
+</head>
+<body>
+<h1>All drivers page</h1>
+<a href="${pageContext.request.contextPath}/">Back to main page</a><br>
+<table border="1">
+    <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>License number</th>
+    </tr>
+    <c:forEach var="driver" items="${drivers}">
+        <tr>
+            <td>
+                <c:out value="${driver.id}"/>
+            </td>
+            <td>
+                <c:out value="${driver.name}"/>
+            </td>
+            <td>
+                <c:out value="${driver.licenseNumber}"/>
+            </td>
+            <td><a href="${pageContext.request.contextPath}/drivers/delete?id=${driver.id}">Delete</a></td>
+        </tr>
+    </c:forEach>
+</table>
+<td><a href="${pageContext.request.contextPath}/drivers/add">Add driver</a></td>
```

Comment:
is it ok that this is outside of the `table` tag?

Reply:
Moved it inside the table, table looks better now.Thank you